### DISCUSSION TITLE
Regions Unexplored

### DISCUSSION THREAD

**Discussion by supermassimo** [DynamicTreesTeam/DynamicTrees#900] [Compatibility Addon Suggestions]
https://www.curseforge.com/minecraft/mc-mods/regions-unexplored

70+ biomes
Hundreds of trees

**Comment by MarijnIsN00B** [comment]
This would be really nice, I hope I can use Dynamic Trees in my modpack one day

**Comment by MoonsMess** [comment]
Would love this too

**Comment by supermassimo** [comment]
The addon for regions unexplored has been released!
https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-regions-unexplored
